pkgs/by-name/ta/tabby-terminal/fix-argv-prefix-splice.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/app/lib/cli.ts b/app/lib/cli.ts
```

Comment:
I wonder if this is avoidable if we set argv0 to node in the wrapper

Reply:
I would expect it to be, but I tried it out and the argv0 was the same. Maybe this is because of how node.js/electron handle the argv when passing it on to the js program.